rfcs/0002-ipfs-governance.md

```diff
@@ -0,0 +1,215 @@
+
+1. Create a landing page for the group that lists its purpose, audiences, members, responsibilities, products and services. (This is often a landing page on a github repository dedicated to the group or its main product/service)
+2. Add the group to the **[TODO]  Full list of Teams and WGs**, grouped according to primary funnel
+3.  Update documentation and websites as necessary, routing audiences to the Team/WG's landing page
+
+### 4. Functioning
+
+Teams are responsible for:
+
+- Defining  [Objectives and Key Results](https://weekdone.com/resources/objectives-key-results) (OKRs) every Quarter. Members of a Product Group primarily derive their tasks from those OKRs.
+- Keeping their landing page up to date, with accurate listings of their Products, Services, Roadmaps, purpose, and members.
+
+Captains are responsible for:
+- Stewarding the creation of OKRs, freezing them, and getting them scored at mid-quarter and end-of-quarter.
+
+### 5. Dissolving or Merging
+If a Team's membership drops below 2 Full Time people, it will be marked "understaffed" and must either add more people, merge with another WG, or dissolve before the next round of OKR planning.
+
+If a Team or WG merges with another Team/WG, the merged group:
+- Does not have to modify its landing page (can still present itself to the world as a distinct group)
+- Incorporates its OKRs into the new group, participating in that group's OKR planning
+- Participates in the new group's planning and coordination activities (weekly check-ins, retrospectives, OKR scoring, etc.)
+- Can un-merge in the future, without submitting a new RFC, if it accumulates enough members to stand on its own.
+
+When a Team or WG is dissolved, the people dissolving it must:
+1. Update its landing page to declare that the it has been dissolved. Reroute visitors accordingly.
+2. Remove the it from the **[TODO] Full list of WGs**  
+3. Remove it from documentation and websites, routing audiences elsewhere
+
+
+## Failsafes
+
+### First Failsafe: Core Working Group
+In case of process failure or broken protocols among the teams, the Core Working Group is responsible for addressing the situation. Their powers and responsibilities are spelled out in the section titled [The Core Working Group](#the-core-working-group).
+
+### "Snakes on the Plane" Failsafe: BDFL
+
+If all other processes and protocols fail, @jbenet will step in and exercise his [beneficially dictatorial](https://en.wikipedia.org/wiki/Benevolent_dictator_for_life) prerogatives as the founder and initial creator of these projects.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+Why should we *not* do this?
+- If we don't set the right protocols for creation & dissolution of working groups, we will end up with a lot of confusion about who is responsible for what, meaning balls will get dropped, people will get frustrated, and productive time will be wasted.
```

Comment:
I may be reading this wrong, but this just seems like another reason why we _**should**_ do this?

Reply:
"Why shouldnt you jump off this cliff?"
"If you don't jump off this cliff, you won't experience the amazing feeling of freefall!"